pages/common/qcp.md

```diff
@@ -0,0 +1,16 @@
+# qcp
+
+> Copy files with your favorite text editor.
+> More information: <https://www.nongnu.org/renameutils/>.
+
+- Copy a single file (invoke an editor with source on the left, target on the right):
+
+`qcp {{source_file}}`
+
+- Copy multiple JPG files:
+
+`qcp {{*.jpg}}`
+
+- Copy files, but swap pattern in the editor -- target on the left and source on the right:
+
+`qcp --option swap {{IMG-*.jpg}}`
```

Comment:
How about remove the preceding `IMG-`, I think `IMG-` does not need here.

Reply:
My bad here, I just happen to remove `IMG-` from `qmv.md`, but forget to remove it from `qcp.md`. Already fix in f9224f2409e35677935e22ed0d19d666a2b182af.